alert-common/src/main/java/com/synopsys/integration/alert/common/channel/issuetracker/service/IssueCreatorTestAction.java

```diff
@@ -90,17 +91,17 @@ public IssueTrackerResponse testConfig(IssueTrackerContext issueTrackerContext)
         String toStatus = "Resolve";
         Optional<String> possibleSecondIssueKey = Optional.empty();
         try {
-            Map<String, String> transitionErrors = new HashMap<>();
+            Map<String, AlertFieldStatus> transitionErrors = new HashMap<>();
```

Comment:
I think it would be a good time to stop using a map here and start using an object that handles all this. It would clear up some code and make it easier to understand.

Reply:
Speaking with Paulo, we think this might be a good change to make as well. Since this is a feature branch we will make these changes in another branch and merge into this one rather than splitting it into two separate PRs.

Reply:
Should `AlertFieldStatus` just have a field for `fieldName`? Then we deal with a list everywhere?